### PR TITLE
UI Fixes :- Change Placeholder for No Data and Add Data

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Change placeholder for Add and No Data
- Fix the Placeholder alignment issue
- Open by default for all policies table in settings

#
### Type of change :

- [x] https://github.com/open-metadata/OpenMetadata/issues/7071#issuecomment-1234330632
- [x] https://github.com/open-metadata/OpenMetadata/issues/7149#issue-1359763048
- [x] [Improvement](https://github.com/open-metadata/OpenMetadata/issues/7071#issuecomment-1234330632)

#
### Frontend Preview (Screenshots) :
image

image

image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@Ashish8689, Checks are failing can you check?

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx [hunk 1]
```diff
@@ -159,10 +159,10 @@ const ActivityFeedList: FC<ActivityFeedListProp> = ({
               tag="button"
               variant="link"
               onClick={() => setFieldListVisible((visible) => !visible)}>
-              <span className="tw-font-medium tw-text-grey">
+              <span className="tw-font-medium tw-text-primary">
                 {feedFilterList.find((f) => f.value === feedFilter)?.name}
               </span>
-              <DropDownIcon />
+              <DropDownIcon style={{ color: '#7147E8' }} />
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx:165]: Can we avoid this inline styling?

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx [hunk 2]
```diff
@@ -183,15 +183,20 @@ const ActivityFeedList: FC<ActivityFeedListProp> = ({
...
+              />
+              <span className="tw-font-medium tw-text-primary tw-ml-1">
                 {
                   threadFilterList.find(
                     (f) => f.value === (threadType ?? 'ALL')
                   )?.name
                 }
               </span>
-              <DropDownIcon />
+              <DropDownIcon style={{ color: '#7147E8' }} />
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx:199]: Same here.

File: openmetadata-ui/src/main/resources/ui/src/components/SampleDataTable/SampleDataTable.component.tsx [hunk 3]
```diff
@@ -153,27 +154,30 @@ const SampleDataTable: FunctionComponent<Props> = ({ sampleData }: Props) => {
...
+              <Typography.Paragraph>
+                {' '}
+                To view Sample Data, run the Profiler Ingestion. Please refer to
+                this doc to schedule the{' '}
+                <Link
+                  className="tw-ml-1"
+                  target="_blank"
+                  to={{
+                    pathname:
+                      'https://docs.open-metadata.org/openmetadata/ingestion/workflows/profiler',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/SampleDataTable/SampleDataTable.component.tsx:175]: Can we move these links to constant and use that constant here?

File: openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx [hunk 4]
```diff
@@ -184,10 +186,23 @@ const Services = ({
...
+          <div className="tw-mt-8 tw-max-w-x tw-text-center">
+            <Paragraph style={{ marginBottom: '4px' }}>
+              {' '}
+              Adding a new {servicesDisplayName[serviceName]} is easy, just give
+              it a spin!
+            </Paragraph>
+            <Paragraph>
+              {' '}
+              Still need help? Refer to our{' '}
+              <AntdLink href="https://ant.design" target="_blank">
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx:200]: Is this link correct?

File: openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx [hunk 5]
```diff
@@ -11,31 +11,90 @@
...
+  heading,
+  buttonLabel,
+  buttonListener,
+  buttons,
+  buttonId,
+}: Props) => {
+  const { Paragraph, Link } = Typography;
+
+  return type === 'ADD_DATA' ? (
+    <div>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx:46]: Is this empty div required?

File: openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx [hunk 6]
```diff
@@ -11,31 +11,90 @@
...
+            </Button>
+          )}
+        </div>
       </div>
-    )}
-  </>
-);
+    </div>
+  ) : (
+    <div>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx:82]: Is this empty div required?

File: openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetail.less [hunk 7]
```diff
@@ -32,10 +32,11 @@
     .ant-collapse-item {
       .ant-collapse-header {
         padding: 16px;
+        align-items: flex-start;
         .ant-collapse-arrow {
           color: @text-color;
           font-size: 14px;
-          margin-right: 6px;
+          padding: 5px 0;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetail.less:39]: Do we need this anymore as we have changed the rule card from collapsible to normal?

File: openmetadata-ui/src/main/resources/ui/src/pages/page-not-found/index.tsx [hunk 8]
```diff
@@ -42,7 +42,7 @@ const PageNotFound = () => {
                 </Button>
               </Link>
               <Link to={ROUTES.EXPLORE}>
-                <Button className="tw-opacity-40" theme="primary">
+                <Button theme="primary" variant="outlined">
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/page-not-found/index.tsx:45]: Can we use `antd` button here?

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx:165]: @Sachin-chaurasiya  This dropdown icon is a generic component and default color is black, so as per our need for our component we have to pass it.

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx:199]: @Sachin-chaurasiya  same for here to.

File: openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx (hunk 5 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx:46]: @Sachin-chaurasiya  It's for the sack of single component return, will use Fragment here instead.

File: openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx (hunk 6 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx:82]: @Sachin-chaurasiya  same for here to.

File: openmetadata-ui/src/main/resources/ui/src/pages/page-not-found/index.tsx (hunk 8 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/page-not-found/index.tsx]: Updated

File: openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetail.less (hunk 7 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetail.less]: Updated

File: openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx (hunk 4 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx]: Updated

File: openmetadata-ui/src/main/resources/ui/src/components/SampleDataTable/SampleDataTable.component.tsx (hunk 3 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/SampleDataTable/SampleDataTable.component.tsx]: Updated

**Pull request closed without merging**